src/middlewared/middlewared/plugins/replication.py

```diff
@@ -439,6 +439,37 @@ async def run(self, job, id, really_run):
 
         await self.middleware.call("zettarepl.run_replication_task", id, really_run, job)
 
+    @accepts(
+        Patch(
+            "replication_create",
+            "replication_run_onetime",
+            ("rm", {"name": "name"}),
+            ("rm", {"name": "auto"}),
+            ("rm", {"name": "schedule"}),
+            ("rm", {"name": "only_matching_schedule"}),
+            ("rm", {"name": "enabled"}),
+        ),
+    )
+    @job(logs=True)
+    async def run_onetime(self, job, data):
```

Comment:
run_once might read better, just a suggestion

Reply:
We already have `cloudsync.sync_onetime` :(